src/frontend/js/index.js

```diff
@@ -956,52 +834,44 @@ const menuView = state => {
                         setTimeout(_ => e.target.innerText = "Copy link", 3000)
 
                     }}, "Copy link"),
-                    state.showctrl && [
+                    state.showctrl && state.meta.scenarios && [
                         m('br'),
 
+                        m('label', {for: 'matrix_variable'}, "Variable"),
+                        m('div[style=display:flex;align-items:center]', [
+                            m('select', {
+                                name: 'matrix_variable',
+                                onchange: e => actions.changeLayerVariable("od_matrices", e.target.value),
+                            },
+                                m('option', {value: '', selected: state.layers.od_matrices.variable === null}, 'None'),
+                                meta2options(state.meta.od_matrices, state.layers.od_matrices.variable)
+                            ),
+                            (state.layers.od_matrices.variable !== "") && [
```

Comment:
```suggestion
                            false && (state.layers.od_matrices.variable !== "") && [
```

Since the route doesn't exist yet